src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -0,0 +1,254 @@
+                </aura:if>
+
+                <!--Template Container-->
+
+                <!-- Step 1: Overview -->
+                <aura:if isTrue="{!v.templateMetadata.progressIndicatorStep == '1' }">
+                    <lightning:layout horizontalAlign="center">
+                        <lightning:layoutItem size="10">
+                            <lightning:input type="text" name="name" label="{!$Label.c.stgLabelName}"
+                                             value="{!v.templateInfo.name}" readonly="{!v.isReadOnly}" required="true"/>
+                            <lightning:textarea name="description" label="{!v.templateMetadata.labels.description}"
+                                                value="{!v.templateInfo.description}" readonly="{!v.isReadOnly}"
+                                                required="true" class="slds-p-top_medium"/>
+                            <div class="slds-p-top_large">
+                                <aura:if isTrue="{!v.isReadOnly}">
+                                    <ui:outputCheckbox value="{!v.templateInfo.requireTotalMatch}" class="slds-form-element__control"/>
+                                    <span class="slds-form-element__label">{!v.templateMetadata.labels.totalMatchLabel}</span>
+
+                                    <aura:set attribute="else">
+                                        <div>
+                                            <lightning:input aura:id="totalMatchCheckbox" type="checkbox" class="templateInputCheckbox"
+                                                             label="{!v.templateMetadata.labels.totalMatchLabel}"
+                                                             name="requireTotalMatch"
+                                                             checked="{!v.templateInfo.requireTotalMatch}"
+                                                             readonly="{!v.isReadOnly}"
+                                                             variant="label-hidden"/>
+                                            <c:CRLP_Tooltip helpText="{!v.templateMetadata.labels.totalMatchHelp}" fieldLabel="{!v.templateMetadata.labels.totalMatchLabel}"/>
+                                        </div>
+                                    </aura:set>
+                                </aura:if>
+                            </div>
+                        </lightning:layoutItem>
+                    </lightning:layout>
+                </aura:if>
+                <!--End Step 1-->
+
+                <!-- Step 2: Template Selection (Only for Batch Gift Entry) -->
+                <aura:if isTrue="{!v.templateMetadata.progressIndicatorStep == '2'}">
+                </aura:if>
+                <!-- End Step 2 -->
+
+                <!-- Step 3: Select Fields -->
+                <aura:if isTrue="{!v.templateMetadata.progressIndicatorStep == '3'}">
+                    <div class="slds-m-around_medium">
+                        <lightning:accordion>
```

Comment:
We should consider replacing this with some basic headers unless we have solid knowledge that this will be fixed in Winter 18.

Reply:
It looks like slds-hide doesn't cause the error in this case, so we may lean on that until we can figure out what's going wrong. The trade-off is that it doesn't open automatically, but we still get the rest of the functionality. We're also still working to reproduce what's going wrong, because a simple repro of it works just fine in other winter orgs.

Reply:
Actually since we only have 2 items, we'll just change these both to be headers that display all available fields and remove the accordions for now.